webapps-backup/src/main/java/io/camunda/webapps/backup/repository/opensearch/OpensearchRequestDSL.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.backup.repository.opensearch;
+
+import io.camunda.webapps.schema.descriptors.IndexTemplateDescriptor;
+import java.util.List;
+import org.opensearch.client.opensearch._types.Time;
+import org.opensearch.client.opensearch.core.GetRequest;
+import org.opensearch.client.opensearch.core.SearchRequest;
+import org.opensearch.client.opensearch.snapshot.CreateSnapshotRequest;
+import org.opensearch.client.opensearch.snapshot.DeleteSnapshotRequest;
+import org.opensearch.client.opensearch.snapshot.GetRepositoryRequest;
+import org.opensearch.client.opensearch.snapshot.GetSnapshotRequest;
+
+/** Copied from operate RequestDSL interface */
+public interface OpensearchRequestDSL {
```

Comment:
Copied from RequstDSL in operate

Reply:
It's smaller - is it because you only copied what it needed?

Reply:
What's the plan for the remaining `RequestDSL`?

Reply:
In general the best thing would be to move it to a common package (but more general than webapps-backup), not sure if we have the time to do it now though